addons/binding/org.openhab.binding.elerotransmitterstick/src/main/java/org/openhab/binding/elerotransmitterstick/stick/CommandUtil.java

```diff
@@ -0,0 +1,102 @@
+/**
+ * Copyright (c) 2010-2017 by the respective copyright holders.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.elerotransmitterstick.stick;
+
+import java.nio.ByteBuffer;
+import java.util.ArrayList;
+
+/**
+ * @author Volker Bier - Initial contribution
+ */
+public class CommandUtil {
+    final protected static char[] hexArray = "0123456789ABCDEF".toCharArray();
+
+    public static String bytesToHex(byte[] bytes) {
```

Comment:
byte/hex conversion is also available in the `Hex` class from Apache Commons Codec library (present in openHAB so it can be used.) so you don't have to write your own.